include/gen_netlink.hrl

```diff
@@ -0,0 +1,235 @@
+-define(ARPHRD_HWX25,    272).            %% Boards with X.25 in firmware
+-define(ARPHRD_CAN,      280).            %% Controller Area Network
+-define(ARPHRD_PPP,      512).
+-define(ARPHRD_CISCO,    513).            %% Cisco HDLC
+-define(ARPHRD_HDLC,     513).
+-define(ARPHRD_LAPB,     516).            %% LAPB
+-define(ARPHRD_DDCMP,    517).            %% Digitals DDCMP protocol
+-define(ARPHRD_RAWHDLC,  518).            %% Raw HDLC
+-define(ARPHRD_TUNNEL,   768).            %% IPIP tunnel
+-define(ARPHRD_TUNNEL6,  769).            %% IP6IP6 tunnel
+-define(ARPHRD_FRAD,     770).            %% Frame Relay Access Device
+-define(ARPHRD_SKIP,     771).            %% SKIP vif
+-define(ARPHRD_LOOPBACK, 772).            %% Loopback device
+-define(ARPHRD_LOCALTLK, 773).            %% Localtalk device
+-define(ARPHRD_FDDI,     774).            %% Fiber Distributed Data Interface
+-define(ARPHRD_BIF,      775).            %% AP1000 BIF
+-define(ARPHRD_SIT,      776).            %% sit0 device - IPv6-in-IPv4
+-define(ARPHRD_IPDDP,    777).            %% IP over DDP tunneller
+-define(ARPHRD_IPGRE,    778).            %% GRE over IP
+-define(ARPHRD_PIMREG,   779).            %% PIMSM register interface
+-define(ARPHRD_HIPPI,    780).            %% High Performance Parallel Interface
+-define(ARPHRD_ASH,      781).            %% Nexus 64Mbps Ash
+-define(ARPHRD_ECONET,   782).            %% Acorn Econet
+-define(ARPHRD_IRDA,     783).            %% Linux-IrDA
+%% ARP works differently on different FC media .. so
+-define(ARPHRD_FCPP,     784).            %% Point to point fibrechannel
+-define(ARPHRD_FCAL,     785).            %% Fibrechannel arbitrated loop
+-define(ARPHRD_FCPL,     786).            %% Fibrechannel public loop
+-define(ARPHRD_FCFABRIC, 787).            %% Fibrechannel fabric
+        %% 787->799 reserved for fibrechannel media types
+-define(ARPHRD_IEEE802_TR, 800).          %% Magic type ident for TR
+-define(ARPHRD_IEEE80211, 801).           %% IEEE 802.11
+-define(ARPHRD_IEEE80211_PRISM, 802).     %% IEEE 802.11 + Prism2 header
+-define(ARPHRD_IEEE80211_RADIOTAP, 803).  %% IEEE 802.11 + radiotap header
+-define(ARPHRD_IEEE802154,         804).
+-define(ARPHRD_PHONET,   820).            %% PhoNet media type
+-define(ARPHRD_PHONET_PIPE, 821).         %% PhoNet pipe header
+-define(ARPHRD_CAIF,     822).            %% CAIF media type
+
+-define(ARPHRD_VOID,       65535).       %% Void type, nothing is known
+-define(ARPHRD_NONE,       65534).       %% zero header length
+
+
+-define(IP_TOS,				1).
+-define(IP_TTL,				2).
+-define(IP_OPTIONS,			4).
+-define(IP_HDRINCL,			3).
+-define(IP_ROUTER_ALERT,		5).
+-define(IP_RECVOPTS,			6).
+-define(IP_RETOPTS,			7).
+-define(IP_PKTINFO,			8).
+-define(IP_PKTOPTIONS,			9).
+-define(IP_PMTUDISC,			10).
+-define(IP_MTU_DISCOVER,		10).
+-define(IP_RECVERR,			11).
+-define(IP_RECVTTL,			12).
+-define(IP_RECVTOS,			13).
+-define(IP_MTU,				14).
```

Comment:
It looks that way, this is code from upstream that's been reformatted / moved around.